### PR TITLE
[Localization] Fix bouncy bubble description for es fr it ko zh

### PR DESCRIPTION
**Pull request by Tempo-anon**
## What are the changes?
Updated bouncy bubble description

## Why am I doing these changes?
It should say 100% of damage dealt as healing instead of 50%

## What did change?
Changed es, fr, it, ko, and both zh localizations for bouncy bubble. Let me know if I've severely butchered anything

### Screenshots/Videos
n/a

## How to test the changes?
Pull

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I tested the changes (manually)?
    - [ ] Are all unit tests still passing? (`npm run test`)
- [ ] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/fr/move.ts [hunk 1]
```diff
@@ -2931,7 +2931,7 @@ export const move: MoveTranslationEntries = {
   },
   "bouncyBubble": {
     name: "Évo-Thalasso",
-    effect: "Évoli frappe l’adversaire avec des bulles d’eau qu’il absorbe ensuite pour récupérer un nombre de PV égal à la moitié des dégâts infligés à l’ennemi."
+    effect: "Évoli frappe l’adversaire avec des bulles d’eau qu’il absorbe ensuite pour récupérer un nombre de PV égal à la des dégâts infligés à l’ennemi."
```
Comment (flx-sta) [inline src/locales/fr/move.ts:2934]: this one still contains the `Evoli` instead of a more generic word

**Review by flx-sta — APPROVED** [review]
Apart from that 1 tiny comment duck-duck-go translator says its good

File: src/locales/fr/move.ts [hunk 2]
```diff
@@ -2931,7 +2931,7 @@ export const move: MoveTranslationEntries = {
   },
   "bouncyBubble": {
     name: "Évo-Thalasso",
-    effect: "Évoli frappe l’adversaire avec des bulles d’eau qu’il absorbe ensuite pour récupérer un nombre de PV égal à la moitié des dégâts infligés à l’ennemi."
+    effect: "Évoli frappe l’adversaire avec des bulles d’eau qu’il absorbe ensuite pour récupérer un nombre de PV égal aux dégâts infligés à l’ennemi."
```
Comment (flx-sta) [inline src/locales/fr/move.ts:2934]:
```suggestion
    effect: "L’adversaire est frappé par des bulles d’eau qui sont ensuite absorbées pour récupérer un nombre de PV égal aux dégâts infligés à l’ennemi."
```

Also from swain but without the Evoli part

**Review by flx-sta — CHANGES_REQUESTED** [review]

File: src/locales/ko/move.ts [hunk 3]
```diff
@@ -2937,7 +2937,7 @@ export const move: MoveTranslationEntries = {
   },
   bouncyBubble: {
     name: "생생버블",
-    effect: "물덩어리를 부딪쳐서 공격한다. 물을 흡수하여 데미지의 절반만큼 HP를 회복한다."
+    effect: "물덩어리를 부딪쳐서 공격한다. 물을 흡수하여 데미지의 절만큼 HP를 회복한다."
```
Comment (jts04) [inline src/locales/ko/move.ts:2940]: wrong change here.
original text is right.

Comment (Tempo-anon) [reply src/locales/ko/move.ts:2940]: Bouncy bubbles heals 100% of the damage dealt in Pokerogue and generations 8 and 9

**Review by Greenlamp2 — APPROVED** [review]

File: src/locales/ko/move.ts (hunk 3 above)
Comment (EnochG1) [reply src/locales/ko/move.ts:2940]: There is no word '절만큼' in korean.
If you want to explain 100% percent of damage, you can fix it like this

```suggestion
    effect: "물덩어리를 부딪쳐서 공격한다. 물을 흡수하여 데미지만큼 HP를 회복한다."
```

**Pull request merged**